src/commitm.zsh

```diff
@@ -147,7 +155,7 @@ commitm() {
         local git_changes=$(cat "$git_output_temp_file")
         
         # Prepare the system prompt with modifications and git changes
-        local full_system_prompt="$system_prompt$prompt_mod"
+        local full_system_prompt="$system_prompt$prompt_mod$prompt"
```

Comment:
@ellipsis-dev it should replace the default prompt, not add onto it.

if the user specifies a custom prompt, you may need to add some text at the end to incorporate the prompt mod.

please fix